### PR TITLE
Chore/readme update

### PR DESCRIPTION
**Pull request by holygits**

### CODE REVIEW

**Comment by holygits** [comment]
test fail unrelated

File: README.md [hunk 1]
```diff
@@ -10,6 +10,28 @@ Manages a Generic Asset - ERC20 peg using the CENNZnet ethereum bridge.
 Deposited ERC20 tokens are claimable on CENNZnet for GA equivalents.  
 Withdrawing GA equivalents requires a signed proof from CENNZnet validators.  
 
+## Dev Setup
+start a CENNZnet validator node.  
+```bash
+cennznet \
```
Comment (Amy-Centrality) [inline README.md:16]: Hmm is this a CENNZnet command line tool? or a shorthand for a docker command?

Comment (Amy-Centrality) [reply README.md:16]: Oh I see, this is running a local build of the CENNZnet repo, is it possible to use a Docker command instead?

**Review by Amy-Centrality — APPROVED** [review]
Looks good :D

**Pull request merged**